packages/tools/src/types/SegmentationStateTypes.ts

```diff
@@ -107,6 +107,13 @@ export type ToolGroupSpecificRepresentationState = {
    * using to render
    */
   colorLUTIndex: number;
+  /**
+   * Poly Seg generated
```

Comment:
Can you spell out in comment that this is a polymorphic segmentation?  At least if you get some information in the type definition it means it isn't a "polygon" or other "poly" type.